views/pages/search.html

```diff
@@ -39,6 +39,19 @@
       {{#input-text}}surname{{/input-text}}
       {{#input-text}}forenames{{/input-text}}
       {{#input-text}}dob{{/input-text}}
+      <details{{#errors.dob}} open{{/errors.dob}}>
+        <summary><span class="summary">You can enter the date in any of these formats:</span></summary>
```

Comment:
Would this be better phrased as: "Which date formats are supported?"

Reply:
I think we should do this, I'll update the text...